torch/_torch_docs.py

```diff
@@ -7980,8 +7980,10 @@ def merge_dicts(*dicts):
 Args:
     input (Tensor): the tensor to narrow
     dim (int): the dimension along which to narrow
-    start (Tensor or int): the starting dimension
-    length (int): the distance to the ending dimension
+    start (int or Tensor): index to narrow from, within `dim`. Can be negative,
```

Comment:
"index of the element to start the narrowed dimension from. Can be negative, which means..."?